### ISSUE TITLE
Make search "order" save selection in config

### ISSUE THREAD

**Issue by bnvk** [mailpile/Mailpile#1527]
Currently, the order messages in a search show up is by default set to `rev-freshness` and upon changing in the UI the new setting does not persist. This can be changed to other states via the CLI using `set prefs.default_order = rev-date`  however, like display density this action should save state in the config via an AJAX call.

**Comment by BjarniRunar** [comment]
This is on my list, but I am unsure what is the best behaviour here.

Changing the default sort order is one thing, which power-users may want to do. However, casual reordering is usually contextual, something you do for the search you are working with - and you may not *want* it to persist. I am not sure we would actually be helping our users by making this sticky.

**Comment by bnvk** [comment]
> reordering is usually contextual, something you do for the search you are working with - and you may not want it to persist. I am not sure we would actually be helping our users by making this sticky.

That definitely makes sense with "casual re-ordering" especially with sorting by oldest first during a focused search. But during normal daily use "show me my Inbox" that would indeed be jarring if someone switched the order and didn't realize it persisted.

Maybe it's just that I found the "freshness" a bit jarring as well- I tried it for a week but really disliked it to due to my mental model of an inbox and keeping track of information visually. Perhaps this is a setting in Gmail or other clients that other people like, i've just never experienced that type of sorting with email clients by default. Suggestions:

- Separate more "search" oriented ordering from other types
- Show an alert / confirm box "remember this order"
- Or needs to be tucked away in some other "Settings" area
